pkg/cli/update.go

```diff
@@ -72,57 +78,5 @@ func (s *Update) Run(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-
-	imageForFlags := image
-	if imageForFlags == "" {
-		imageForFlags = app.Spec.Image
-
-		if _, isPattern := autoupgrade.AutoUpgradePattern(imageForFlags); isPattern {
-			imageForFlags = app.Status.AppImage.ID
-		}
```

Comment:
I don't see this logic being added back anywhere. This is to allow passing Acorn args with images with autouprade image tags.

Reply:
`	_, flags, err := deployargs.ToFlagsFromImage(cmd.Context(), c, imageForFlags)`

was returning the right image id from the app.spec.imge being `test:v#.#.#` 

I did `acorn run test:v#.#.#` which pulled used `test:v0.0.4` I then did `acorn update -n app_name -l label=new -- --msg 12` 

and it seemed to get both of those args in the update. but I added it in in case I'm missing a usage